src/gettext_boost.cpp

```diff
@@ -425,7 +425,13 @@ int compare(const std::string& s1, const std::string& s2)
 
 int icompare(const std::string& s1, const std::string& s2)
 {
-	return std::use_facet<bl::collator<char> >(get_manager().get_locale()).compare(bl::collator_base::secondary, s1, s2);
+#ifdef __APPLE__
+	// https://github.com/wesnoth/wesnoth/issues/1759
```

Comment:
This looks like #2094, not #1759. Please adjust the comment and the commit message.